v5/Http/Resources/PostResource.php

```diff
@@ -123,6 +123,16 @@ private function hydrateResourceRelationships($request)
         return $result;
     }
 
+    private function getPostSource()
+    {
+        $source = "web";
```

Comment:
I would suggest using setting the web as a const in the top of class declaration so the context of what 'web' is is clear.

```php
class PostResource
{
    private const DEFAULT_SOURCE_TYPE = 'web';
    ...
```